addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -2766,3 +2766,26 @@ export function cleanZWS(node) {
         .filter(node => node.nodeType === Node.TEXT_NODE)
         .forEach(node => node.nodeValue = node.nodeValue.replace(/\u200B/g, ''));
 }
+
+/**
+ * Return true if click or mousedown event occurred inside the checkbox box.
+ * False otherwise.
+ *
+ * @param {Node} node
+ * @returns {boolean}
+ */
+export function isMouseInsideCheckbox(ev) {
+    const node = ev.target;
+    const beforeStyle = window.getComputedStyle(node, ':before');
+    const style1 = {
```

Comment:
Actually also while you're at it, maybe you could rename this to `style`. I don't get why it was called `style1` if there's no `style2`.